### PR TITLE
ci: incorporate more style check and package build elements from Biocommons template 

### PR DESCRIPTION
**Pull request by jsstevenson**
Progress on #23 

Includes a `black` run, so lots of annoying diffs. Also moves more things over to `pyproject.toml`. Adds some GitHub cookie cutter things.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (korikuzma) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]:
```suggestion
labels: 'bug'
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Additional context**
+Add any other context about the problem here.
```
Comment (korikuzma) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: Do we want to do the thing where they confirm "yes i've checked that an issue does not already exist"

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (korikuzma) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]:
```suggestion
labels: 'enhancement'
```

File: .github/workflows/python-package.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  cqa:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
```
Comment (korikuzma) [inline .github/workflows/python-package.yml:17]: We should use the python matrix for other versions

**Review by korikuzma — COMMENTED** [review]
initial review before lunch

**Comment by jsstevenson** [comment]
@korikuzma I'm a little hesitant to introduce more differences between this and the template? or maybe it doesn't matter tbh

**Review by korikuzma — APPROVED** [review]

**Pull request merged**